opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraphBar.jsx

```diff
@@ -865,6 +870,13 @@ class InvestigationGraphBar extends Component {
                           />
                         </>
                       )}
+                      <Tooltip title={t('Restore the state of the graphic before the last expansion')}>
+                        <span>
+                          <IconButton color="primary" onClick={handleOpenRollBackToPreExpansionStateDialog} size="large" disabled={isRollBackToLastPreExpansionStateEnabled}>
```

Comment:
I would print one prop per line for readability, maybe it's not covered by lint.